lib/project_types/script/messages/messages.rb

```diff
@@ -164,6 +164,9 @@ module Messages
           missing_env_file_variables: "The following are missing in the .env file: %s. ",
           missing_env_file_variables_solution: "To add it, connect your script with " \
           "{{command:%1$s script connect}} ",
+
+          invalid_app_bridge_path_cause: "The App Bridge %{path_type} path is invalid.",
+          invalid_app_bridge_path_help: "Set the app_bridge_%{path_type}_path in .shopify-cli.yml to a relative App Bridge path.",
```

Comment:
How about: The script couldn't be pushed because the App Bridge path is incorrect in .shopify-cli.yml. Make sure that the app_bridge_create_path is set to a relative path.